ui/guest/src/react/ZoneMarker.tsx

```diff
@@ -179,7 +179,9 @@ export function ZoneMarker(props: ZoneMarkerProps) {
                   onClick={(e) => e.stopPropagation()}
                 >
                   {Object.entries(allowedTypesMeta).map(([id, modes]) => {
-                    const type = contentTypes[id];
+                    // 'id' can be '*' (meaning all content types), so we need to handle that scenario
+                    // TODO: i18n
+                    const type = id === '*' ? { id: 'all', name: 'all types' } : contentTypes[id];
```

Comment:
Where does the `all` id of this on-the-fly object ends up being used? Don't see any other change consuming this ID.

What's the plan for this TODO?
At a minimum this should have proper capitalisation...
```suggestion
                    // 'id' can be '*' (meaning all content types), so we need to handle that scenario
                    // TODO: i18n
                    const type = id === '*' ? { id: 'all', name: 'All types' } : contentTypes[id];
```

Reply:
The 'all' id object is only for the `getAvatarWithIconColors` to get a proper avatar.
We have some TODOS all over guest for when we include i18n to it. Updated for it to be capitalized